src/main/java/com/pet/bankservice/util/HttpClient.java

```diff
@@ -0,0 +1,38 @@
+package com.pet.bankservice.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.pet.bankservice.entity.dto.CurrencyExchangeDto;
+import com.pet.bankservice.exception.ApiUnavailableException;
+import java.io.IOException;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.springframework.stereotype.Component;
+
+@Data
```

Comment:
better use `@AllArgsConstructor` in these cases

Reply:
done, I was inattentive